### PR TITLE
make sure all white space is 2 spaces

### PR DESCRIPTION
**Pull request by danmitchell-**
Clean up frontend code based on code styleguides.

No code changes. 💯

### CODE REVIEW

**Comment by tvararu** [comment]
Looks good! 👏  I'm going to double check to see if there's any other files.

One nit: the [git styleguide](https://github.com/alphagov/styleguides/blob/master/git.md) recommend formatting the first line like an email subject, with a leading capital and no trailing dot.

We also usually add a message describing the "why" of the commit, but that isn't be necessary here.

**Comment by samcrang** [comment]
Does anyone have any strong feels either way about setting up an [EditorConfig](http://editorconfig.org/) for this project?

File: src/main/resources/templates/fragments/entry-table.html [hunk 1]
```diff
@@ -2,39 +2,39 @@
...
-            <td th:text="${entry.timestampAsISOFormat}"></td>
-        </tr>
-        <tr>
-            <td th:text="key"></td>
-            <td th:text="${entry.key}"></td>
-        </tr>
-        </tbody>
-    </table>
+      <colgroup>
+      <col class="field-column"/>
```
Comment (tvararu) [inline src/main/resources/templates/fragments/entry-table.html]: I think this is off by one, and it throws off the rest of the file.

File: src/main/resources/templates/home.html [hunk 2]
```diff
@@ -3,84 +3,82 @@
...
-            <h2 class="heading-large">Technical information</h2>
-                <p>Registers are made up of entries containing fields of data. To build services on top of these registers you can access the data in a variety of formats.</p>
-                <h3>Fields in this register:</h3>
-                <div class="field-list"> <ul><li th:each="field : ${register.fields}" th:text="${field}"></li></ul> </div>
+      <div class="grid-row">
+        <div class="column-two-thirds">
+          <h2 class="heading-large">Technical information</h2>
+          <p>Registers are made up of entries containing fields of data. To build services on top of these registers you can access the data in a variety of formats.</p>
+          <h3>Fields in this register:</h3>
+          <div class="field-list"> <ul><li th:each="field : ${register.fields}" th:text="${field}"></li></ul> </div>
```
Comment (tvararu) [inline src/main/resources/templates/home.html]: I think we can spread this out over multiple lines, it's a looping construct so would be good to increase visibility.

File: src/main/resources/templates/main.html [hunk 3]
```diff
@@ -3,92 +3,92 @@
...
+  <p>
+    <strong class="phase-tag" th:text="${#strings.toUpperCase(register.phase)}">ALPHA</strong>
+    <span>This is a new service – your <a href="mailto:data-infrastructure@digital.cabinet-office.gov.uk">feedback</a> will help us to improve it.</span>
+  </p>
 </div>
 
 <th:block th:fragment="organisation(organisation, branding)">
-    <span th:if="${branding.present}" th:utext="${#strings.replace(branding.get().logoFormattedName,'\r\n','&lt;br/&gt;')}" class="organisation-logo" th:classappend="|organisation-logo-${branding.get().logoClassName} ${branding.get().colourClassName}|">Cabinet Office</span>
-    <span th:unless="${branding.present}" th:text="${organisation.name}" class="organisation-logo organisation-logo-no-identity">Cabinet Office</span>
+<span th:if="${branding.present}" th:utext="${#strings.replace(branding.get().logoFormattedName,'\r\n','&lt;br/&gt;')}" class="organisation-logo" th:classappend="|organisation-logo-${branding.get().logoClassName} ${branding.get().colourClassName}|">Cabinet Office</span>
```
Comment (tvararu) [inline src/main/resources/templates/main.html]: These are off by one I think.

File: src/main/resources/templates/main.html [hunk 4]
```diff
@@ -3,92 +3,92 @@
...
-                </div>
-                <div th:unless="${renderedCopyrightText.present}" class="open-government-licence">
-                    <p class="logo"><a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
-                                       rel="license">Open Government Licence</a></p>
+        <div th:if="${renderedCopyrightText.present}" class="custom-government-licence">
+          <div th:replace="copyright.html::copyright">Copyright text</div>
+        </div>
+        <div th:unless="${renderedCopyrightText.present}" class="open-government-licence">
+          <p class="logo"><a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+           rel="license">Open Government Licence</a></p>
```
Comment (tvararu) [inline src/main/resources/templates/main.html]: We should put the `<a>` here on its own line rather.

File: src/main/resources/templates/main.html [hunk 5]
```diff
@@ -3,92 +3,92 @@
...
-                </div>
-            </div>
-            <div class="copyright">
-                <a href="http://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/copyright-and-re-use/crown-copyright/">©
-                    Crown copyright</a>
-            </div>
+           <p>All content is available under the <a
+            href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+            rel="license">Open
+            Government Licence v3.0</a>, except where otherwise stated</p>
```
Comment (tvararu) [inline src/main/resources/templates/main.html]: Indentation is off here, and I'd put the `</p>` on its own.

**Comment by tvararu** [comment]
@samcrang I'm all for that!

**Pull request closed without merging**